### ISSUE TITLE
The class colours for btn-link appear to be reversed

### ISSUE THREAD

**Issue by rabbit59** [vinorodrigues/bootstrap-dark-5#49]
When using a dark theme the colours for a btn-link appear to be reversed.  I'm not sure whether this is 'by design' or an actual issue.

The colour for a standard bootstrap btn-class goes from light colour to a darker hue, whereas the dark theme goes from a darker colour to a lighter hue.

Please see sample.html in attached sample.zip (also requires iframe-std.html and iframe-dark.html)
[sample.zip](https://github.com/vinorodrigues/bootstrap-dark-5/files/7792021/sample.zip)
 
(PS.  Please feel free to tell me to stop posting issues as I do understand that this is a proof of concept only)

**Comment by vinorodrigues** [comment]
I think you'll find this a problem with just a generic link (`<a href=`) type.

This one also got me when I first ported the code.   See `_variables-alt.scss`, around line 365.

If I had just converted the code by just sufixing the variables with `-alt` it would look like this:
```scss
$link-color-alt:             $primary-alt !default;
$link-shade-percentage-alt:  -20% !default;
$link-hover-color-alt:       shift-color($link-color-alt, $link-shade-percentage-alt) !default;
```

But when doing that the link color is too dark.  I got around that by pre-shifting the `$link-color-alt` variable with:
```scss
$link-color-alt:  shift-color($primary-alt, -15%) !default;
```
And this seems to a an OK (but not perfect) outcome - however, this then causes a compile error on the next two lines as the `shift-color` (and `$link-shade-percentage` variable) causes the "hover" color to be ilegible.

The problem is that the default behaviour darkens the link color when hovered - and for dark mode this would be a problem: either the link would be too bright and then darken on over, or the hover color would be too dark.  I chose to brighten on hover bacaus I just could not see a way around the issue.

The code that's left behind is the outcome of me being hasty in trying to get the code to work within the philosophy of not adding additional code to the port.  I too am not happy with it - but I just don't see a way around the issue without changing core functions to accommodate the transition from light to dark ... i.e. core functions like `shift-color` have not been built to accommodate light/dark variations.

I don't have a answer to this right now.  Maybe experiment with some options here on a fork and test the effects of generic links on all backgrounds (`.btn-link` should look like generic `<a href=` links after all).

> Please feel free to tell me to stop posting issues

Nope, it's OK - I like to think every now and then ... may not have the time sometimes, but I'll give it a go when I can.,
